src/components/Booklist.js

```diff
@@ -1,14 +1,25 @@
-import React from 'react';
-import { useSelector } from 'react-redux';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 import Books from './Books';
 import FormInput from './FormInput';
+import { fetchBooks } from '../redux/books/books';
 
 const Booklist = () => {
   const Booklist = useSelector((state) => state.books);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchBooks(), []);
+  });
```

Comment:
- [ ] What the previous reviewer actually meant is that to fix the error of re-rendering you can fix it in the way he provided. Kindly add the code as he put it.

```
useEffect(() => {
    dispatch(fetchBooks());
  }, [dispatch]); 
```

[Screencast from 21-12-2022 05:07:07 ALASIRI.webm](https://user-images.githubusercontent.com/86318284/208924429-83aca86a-48ba-4f95-ba99-94cd0e6af5ce.webm)

Reply:
@Nemwel-Boniface Oh Sorry. I made a mistake. I actually made the changes and forgot to commit it.